doc/decisions/elektra_web.md

```diff
@@ -0,0 +1,65 @@
+# elektra-web structure
+
+## Issue
+
+For elektra-web, there needs to be a way to remotely manage instances and groups
+of instances (clusters). The remote configuration of a single instance is
+simple. However, to manage multiple instances, we need to store the information
+to access the daemons, as well as information about the grouping (clusters) of
+daemons.
+
+## Constraints
+
+- We need to be able to manage single, as well as multiple instances.
+- We need to be able to group instances and manage them together.
+
+## Assumptions
+
+## Considered Alternatives
+
+- Accessing the elektrad daemons directly
+- Using a separate daemon (clusterd) to manage multiple instances
+  - Using one of these daemons for each cluster
+  - Using one daemon for all clusters
+
+## Decision
+
+Use one cluster daemon (clusterd) to manage all clusters and instances.
+
+## Argument
+
+Accessing the elektrad daemons directly would require us to store all the
+information in the client, which is not a good idea if we want to be able to
+switch machines and still be able to access all data. Thus, there should be a
+single point of entry that the client connects to.
+
+Using a daemon for each cluster would make it harder to create new clusters,
+because the user would have to manually set up a new daemon for each cluster.
+Another problem with this approach is that we still don't have a single point of
+entry, so we would have to store information to connect to the daemons on the
+client.
+
+Using one daemon to manage all instances and clusters is the easiest solution,
+that way the client simply connects to that daemon, which forwards requests to
+single instances or multiple instances at once. In this case, the cluster daemon
+can also serve the client, further simplifying the whole structure, because we
+don't need to host the client on a separate web server.
```

Comment:
Another argument pro single access is: Thus both clusterd and elektrad are nodejs, it does not matter dependency-wise. And a wrapper script can start both daemons easily.

The script should also setup the clusterd to have the elektrad on the same node as only instance added. This solution would be quite optimal.